elements/rh-breadcrumb/rh-breadcrumb.ts

```diff
@@ -0,0 +1,89 @@
+ *         {@default `var(--rh-color-text-primary-on-light, #151515)`}
+ *
+ * @cssprop --rh-breadcrumb-link-color-current-page-subtle
+ *         The current page's color for the `subtle` variant
+ *         {@default `var(--rh-color-text-secondary-on-light, #4d4d4d)`}
+ *
+ * @cssprop --rh-breadcrumb-link-color-hover
+ *         The link color on hover/focus/active for each anchor in the list
+ *         {@default `var(--rh-color-interactive-blue-darkest, #003366)`}
+ *
+ * @cssprop --rh-breadcrumb-link-color-visited
+ *         The visited color for each breadcrumb link
+ *         {@default `var(--rh-color-interactive-purple-darker, #5e40be)`}
+ *
+ * @cssprop --rh-breadcrumb-link-color-visited-hover
+ *         The visited color on hover for each breadcrumb link
+ *         {@default `var(--rh-color-interactive-purple-darkest, #21134d)`}
+ *
+ * @cssprop --rh-breadcrumb-link-focus-outline-color
+ *         The link focus outline color
+ *         {@default `var(--rh-color-border-interactive-on-light, #0066cc)`}
+ *
+ * @cssprop --rh-breadcrumb-li-padding-inline-end
+ *          Sets the spacing between each breadcrumb item.
+ *          {@default `var(--rh-breadcrumb-li-padding-inline-end, 42px)`}
+ *
+ * @cssprop --rh-breadcrumb-caret-image
+ *          The default background image separating each breadcrumb item
+ *          {@default `var(--rh-breadcrumb-caret-image, url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='15' viewBox='0 0 10 15' fill='none'%3E%3Cg clip-path='url(%23clip0_232_19576)'%3E%3Cpath d='M2.5 14.5L0.5 12.5L5.5 7.5L0.5 2.5L2.5 0.5L9.5 7.5L2.5 14.5Z' fill='%23ffffff'/%3E%3C/g%3E%3Cdefs%3E%3CclipPath id='clip0_232_19576'%3E%3Crect width='10' height='14' fill='white' transform='matrix(-1 0 0 1 10 0.5)'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E"))`}
+ *
+ * @cssprop --rh-breadcrumb-caret-image-subtle
+ *          The `subtle` variant background image separating each breadcrumb item
+ *          {@default `var(--rh-breadcrumb-caret-image-subtle, url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='15' viewBox='0 0 10 15' fill='none'%3E%3Cg clip-path='url(%23clip0_232_19576)'%3E%3Cpath d='M2.5 14.5L0.5 12.5L5.5 7.5L0.5 2.5L2.5 0.5L9.5 7.5L2.5 14.5Z' fill='%23a3a3a3'/%3E%3C/g%3E%3Cdefs%3E%3CclipPath id='clip0_232_19576'%3E%3Crect width='10' height='14' fill='white' transform='matrix(-1 0 0 1 10 0.5)'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E"))`}
+ *
+*/
+
+@customElement('rh-breadcrumb')
+export class RhBreadcrumb extends LitElement {
+  @property({ type: Array }) items: string[] = [];
+
+  @property({ reflect: true }) variant?: 'subtle';
+
+  /**
+   * Sets color theme based on parent context
+   */
+  @colorContextConsumer() private on?: ColorTheme;
+
+  static readonly styles = [styles];
+
+  render() {
+    const { on = '' } = this;
+    return html`
+      <nav aria-label="Breadcrumb" part="container" id="container" class="${classMap({ [on]: !!on })}">
```

Comment:
trouble for l10n

Reply:
maybe it's better to apply the nav role to the host and require a label there

Reply:
I added an [optional `accessible-label` property](https://github.com/RedHat-UX/red-hat-design-system/pull/1535/commits/92b41ba507a89dfc1554cb5817e011f7034b5a1d), eg:

```html
<rh-breadcrumb accessible-label="Breadcrumbs label goes here">
  ...markup goes here...
</rh-breadcrumb>
```

If a user doesn't specify an `accessible-label`, it will fall back/default to "Breadcrumb".